### ISSUE TITLE
pagmo-devel 2.19 build 3 not found by meson build system

### ISSUE THREAD

**Issue by lbocklag** [esa/pagmo2#557]
pagmo-devel 2.19 build 3 is not recognized by the meson build system although it is installed  from confa forge.

Meson message:
Run-time dependency pagmo found: NO (tried pkgconfig and cmake)
src/nexus/clib/meson.build:26:12: ERROR: Dependency "Pagmo" not found, tried pkgconfig and cmake

Pagmo 2.18 and 2.19 build 2 were working.

 - OS: Linux
 - Installation method: conda forge
 - Version: 2.19 build 3

**Comment by lbocklag** [comment]
Thanks @bluescarni

Using boost-cpp 1.82 works for pagmo-devel 2.19 build 3 and 4.
The packages however download libboost-devel instead of boost-cpp, and are not recognized by the meson build system.

The changes mentioned in https://github.com/conda-forge/pagmo-feedstock/pull/71 on [recipe/meta.yaml](https://github.com/conda-forge/pagmo-feedstock/pull/71/files/1d57b9650cb6ad3963934a02b4806e1087cbb5c7#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9a) removed boost-cpp.

**Comment by bluescarni** [comment]
@lbocklag I think there was recently some conda-forge-wide renaming involving the Boost package, but I am not familiar with the details.

Pinging @h-vetinari as an authority on the matter.

**Comment by h-vetinari** [comment]
> The packages however download libboost-devel instead of boost-cpp, and are not recognized by the meson build system.

This is correct (the output got renamed) and should be functionally equivalent to what boost-cpp provided. In particular, there's all the necessary cmake metadata, and I think pkgconfig as well (though I have to double-check on the latter)

**Comment by h-vetinari** [comment]
So meson should keep finding boost.

**Comment by jschueller** [comment]
likely fixed by https://github.com/conda-forge/pagmo-feedstock/pull/80 btw

**Comment by darioizzo** [comment]
closing
